projects/ccf-shared/src/lib/components/organ-selector/organ-selector.component.ts

```diff
@@ -1,12 +1,12 @@
 import { AfterViewInit, OnDestroy, Component, EventEmitter, HostBinding, Input, Output, ElementRef, ViewChild } from '@angular/core';
 import { ResizeSensor } from 'css-element-queries';
 
-export const ALL_ORGANS: OrganInfo[] = [
-  { src: 'app:large_intestine', organ: 'Large Intestine', name: 'Large Intestine', hasSex: true },
-  { src: 'app:heart', organ: 'Heart', name: 'Heart', hasSex: true },
-  { src: 'app:kidney-left', organ: 'Kidney', name: 'Kidney, L', side: 'left', hasSex: true },
-  { src: 'app:kidney-right', organ: 'Kidney', name: 'Kidney, R', side: 'right', hasSex: true },
-  { src: 'app:spleen', organ: 'Spleen', name: 'Spleen', hasSex: true },
+export let ALL_ORGANS: OrganInfo[] = [
```

Comment:
This can still be const by adding the filter at the end of the ].. Ie `[foo].filter(x => x.blah)`.. Also if you test for `organ.disabled !== true` no need for setting `disabled: false`.

Reply:
I had done that initially but I kept getting this,
```js
let ALL_ORGANS: OrganInfo[]
Type '({ src: string; organ: string; name: string; hasSex: boolean; side?: undefined; disabled?: undefined; } | { src: string; organ: string; name: string; side: string; hasSex: boolean; disabled?: undefined; } | { ...; } | { ...; })[]' is not assignable to type 'OrganInfo[]'.
  Type '{ src: string; organ: string; name: string; hasSex: boolean; side?: undefined; disabled?: undefined; } | { src: string; organ: string; name: string; side: string; hasSex: boolean; disabled?: undefined; } | { ...; } | { ...; }' is not assignable to type 'OrganInfo'.
    Type '{ src: string; organ: string; name: string; side: string; hasSex: boolean; disabled?: undefined; }' is not assignable to type 'OrganInfo'.
      Types of property 'side' are incompatible.
        Type 'string' is not assignable to type '"left" | "right" | undefined'.ts(2322)
```

which didn't make sense cause I know filter returns an array

Reply:
I think you just add as OrganInfo[] after the filter function.. ie `[].filter() as OrganInfo[]` ?